### PR TITLE
Image linking props

### LINKED ISSUE #1507: info.json should allow seeAlso for discovery of image content
Use Case:  If an institution has images with metadata, but the metadata is not IIIF Presentation API, then their images cannot be discovered.  Instead, if the info.json allowed a reference via `seeAlso` to an external metadata format, then this could be used directly for the labels and descriptive metadata.  This doesn't muddy the water of presentation API's relationship with image, nor of what is in scope for info.json, it just allows a reference to an external document that may or may not be useful for a client to know about.

Towards discovery, this would decouple image and presentation for the purposes of finding descriptive information about the image, and allow additional notifications about image content becoming available separately from Manifests that reference that content. 

Proposal:

Import `seeAlso` from prezi3 into info.json in image3.

/ht Randy Stern

### PR DESCRIPTION
**Pull request by mikeapp**
Closes #1507

### CODE REVIEW

File: source/api/image/3.0/index.md [hunk 1]
```diff
@@ -599,7 +599,7 @@ The rights and licensing properties, `requiredStatement`, `rights` and `logo`, h
...
+| `logo` | Optional | A small external image resource that represents an individual or organization associated with this image. This could be the logo of the owning or hosting institution. The value of this property _MUST_ be an array of JSON objects, each of which _MUST_ have an `id` and should have at least one of `type` and `format`. The logo _MUST_ be clearly rendered when the resource is displayed or used, without cropping, rotating or otherwise distorting the image. It is _RECOMMENDED_ that a IIIF Image API [service](#linking-properties) be available for this image for other manipulations such as resizing.  |
```
Comment (azaroth42) [inline source/api/image/3.0/index.md]: _SHOULD_

File: source/api/image/3.0/index.md [hunk 2]
```diff
@@ -667,31 +667,54 @@ A server that supports neither `sizeByW` or `sizeByWh` is only required to serve
...
-### 5.7. Related Services
+### 5.7. Linking Properties
 
-The JSON response _MAY_ also reference external services that make additional functionality available to a viewer.
+The JSON response _MAY_ contain linking properties that reference external resources, including services that make additional functionality available to a viewer.  The linking properties have the same semantics and requirements as those in the [Presentation API][prezi-api].
 
 | Property   | Required? | Description |
 | ---------- | --------- | ----------- |
-| `service`  | Optional  | The `service` references external services that the client might interact with directly to gain additional information or functionality, for example a link to an authentication service. The value _MUST_ be a list of objects. See the [Service Registry][service-profiles] for known service types. |
+| `partOf`   | Optional  | A link to another resource that contains this image service, for example a link to a Canvas or Manifest. The value _MUST_ be a list of objects. Each item _MUST_ have the `id` and `type` properties, and _SHOULD_ have the `label` property. |
```
Comment (azaroth42) [inline source/api/image/3.0/index.md]: contains or references?

Comment (zimeon) [reply source/api/image/3.0/index.md]: I think the Canvas or Manifest _contains_ -- that usefully distinguishes from the linking to the Canvas or Manifest

File: source/_includes/links.md [hunk 3]
```diff
@@ -116,6 +116,7 @@
 [prezi3-range]: {{ site.url }}{{ site.baseurl }}/api/presentation/3.0/#range
 [prezi3-resource-structure]: {{ site.url }}{{ site.baseurl }}/api/presentation/3.0/#resource-structure
 [prezi3-terminology]: {{ site.url }}{{ site.baseurl }}/api/presentation/3.0/#terminology
+[prezi3-type]: {{ site.url }}{{ site.baseurl }}/api/presentation/3.0/#type
```
Comment (zimeon) [inline source/_includes/links.md]: prezi30-type

File: source/api/image/3.0/index.md [hunk 4]
```diff
@@ -667,31 +667,54 @@ A server that supports neither `sizeByW` or `sizeByWh` is only required to serve
 
 The set of features, formats and qualities supported is the union of those declared in the external profile document and those added by the `extra` properties. If a feature is not present in either the profile document or the `extraFeatures` property, then a client _MUST_ assume that the feature is not supported.
 
-### 5.7. Related Services
+### 5.7. Linking Properties
 
-The JSON response _MAY_ also reference external services that make additional functionality available to a viewer.
+The JSON response _MAY_ contain linking properties that reference external resources, including services that make additional functionality available to a viewer.  The linking properties have the same semantics and requirements as those in the [Presentation API][prezi-api].
```
Comment (azaroth42) [inline source/api/image/3.0/index.md]: --> prezi3-api

**Pull request merged**